src/completeTasks.js

```diff
@@ -0,0 +1,24 @@
+// eslint-disable-next-line import/no-cycle
```

Comment:
- [ ]  [optional] please remove all the unused commented code in order to keep the file clean and pro